src/main/java/com/conveyal/gtfs/loader/JdbcGtfsLoader.java

```diff
@@ -516,7 +524,7 @@ public void setValueForField(Table table, int fieldIndex, int lineNumber, Field
                     ValidateFieldResult<String> result = field.validateAndConvert(string);
                     // If the result is null, use the null-setting method.
                     if (result.clean == null) setFieldToNull(postgresText, transformedStrings, fieldIndex, field);
-                    // Otherwise, set the cleaned field according to its index.
+                        // Otherwise, set the cleaned field according to its index.
                     else transformedStrings[fieldIndex + 1] = result.clean;
```

Comment:
It'd probably look nice to format this section with brackets around the if/else blocks.

Reply:
Updated.